src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -259,13 +266,19 @@ public override int Execute()
 
                     Debug.Assert(result != 0);
                 }
+
+                Debug.Assert(buildLevel is BuildLevel.All or BuildLevel.Csc);
             }
 
             MarkBuildStart();
         }
-        else
+        else // if (NoBuild)
```

Comment:
nit: commented code

Reply:
That's not commented code, that's my attempt at making it clearer what "state" the `else` branch is in since it's a bit far away from the `if (!NoBuild)` above.